tools/cldr/cldr-to-icu/build-icu-data.xml

```diff
@@ -332,6 +335,18 @@
         </convert>
     </target>
 
+    <target name="copy-cldrtestdata" depends="init-args">
+        <copy todir="${testDataDir}/cldr-testData">
+            <fileset dir="${cldrDir}/common/testData">
+                <include name="units/**"/>
```

Comment:
Is this where you want people to add more directories?
If so, please comment as such.
Maybe also comment what the inclusions are intended for (so one day some could even be removed).

Reply:
Done.